cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -123,6 +123,7 @@ resources:
 - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/moc-nfs-democratic-csi-node-rb
 - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/node-labeler
 - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/oauth-token-access-review-opf-observatorium
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/reloader-reloader-role-binding
```

Comment:
Can you also add the namespace in this file that you created in base? (alphabetically)

Reply:
@HumairAK Oops missed that. Added